docs/ssss.md

```diff
@@ -1,3 +1,106 @@
 # Simperby Super Simple Summary
 
-TODO
+## DAO
+
+A DAO (Decentralized Autonomous Organization) is an organization that operates
+autonomously on a blockchain without relying on government or centralized
+control. It is governed by a set of rules encoded in the protocol, and is based
+on a trustless network where all decisions are recorded and verified by the
+network participants.
+
+For more information on DAOs, refer to [the introduction to DAO
+(WIP)](./dao.md).
+
+## How to Implement a DAO
```

Comment:
I guess this title is a bit going off point. This section is telling about the limitation and disadvantages of typical DAOs.

Reply:
agree with this

Reply:
changed as we discussed in Discord